### PR TITLE
update: separate target set up removed

### PR DESCRIPTION
**Pull request by danil-pavlov**

### CODE REVIEW

File: docs/topics/multiplatform/multiplatform-dsl-reference.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ Inside `kotlin {}`, you can write the following blocks:
 
 A _target_ is a part of the build responsible for compiling, testing, and packaging a piece of software aimed at
 one of the supported platforms. Kotlin provides targets for each platform, so you can instruct Kotlin to compile code for
-that specific target. To learn more about how to configure these targets, see [Set up targets for Kotlin Multiplatform](multiplatform-set-up-targets.md).
+that specific target. To learn more about how to configure these targets. See how to [set up a target](multiplatform-discover-project.md#targets).
```
Comment (sarahhaggarty) [inline docs/topics/multiplatform/multiplatform-dsl-reference.md:50]: Did you mean to delete "To learn more about how to configure these targets." ? 🤔

File: docs/topics/native/native-target-support.md
Comment (sarahhaggarty) [inline docs/topics/native/native-target-support.md:1]: Should we also remove the mentions of "target presets" on lines 80 and 82? 🤔

**Review by sarahhaggarty — CHANGES_REQUESTED** [review]
Looks good! There's just one sentence that I'm not sure about and some other mentions of "target presets" that would be good to double-check :)

File: docs/topics/multiplatform/multiplatform-dsl-reference.md (hunk 1 above)
Comment (danil-pavlov) [reply docs/topics/multiplatform/multiplatform-dsl-reference.md]: Yeah, missed extra sentence

**Pull request merged**